Multisig/Cross-layer/App.swift

```diff
@@ -27,6 +27,18 @@ class App {
     logger: LogService.shared)
 
     var nodeService = EthereumNodeService(url: URL(string: "https://rinkeby.infura.io/v3/438e11915f8b4834a05e7810b88db4b3")!)
+
+    let termOfUseURL = URL(string:"https://gnosis-safe.io/terms/")!
+    let privacyPolicyURL = URL(string:"https://gnosis-safe.io/privacy/")!
+    let licensesURL = URL(string:"https://gnosis-safe.io/licenses/")!
+    let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ?? ""
```

Comment:
I would make the version string of the format "CFBundleShortVersionString (CFBundleVersion)"

Reply:
like 0.1.0 (2)

Reply:
This comment was not addressed